src/lib/components/Button/Button.tsx

```diff
@@ -124,6 +143,13 @@ const ButtonComponent = forwardRef<HTMLButtonElement | HTMLAnchorElement, Button
           )}
         >
           <>
+            {isProcessing && (
+              <Spinner
```

Comment:
BTW, I will also allow the user to overwrite the spinner component here, if not, use the default one.